src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [product, setProduct] = useState('Jam');
+  const NO_GOODS_SELECTED = 'No goods';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {product}
+        {product !== NO_GOODS_SELECTED ? ' is ' : ' '}
```

Comment:
it seems more complicated than just checking if any product is selected and rendering 2 different messages instead of building it from parts as you do